django_tenants/management/commands/migrate_schemas.py

```diff
@@ -14,6 +15,7 @@
 
 
 class MigrateSchemasCommand(SyncCommon):
+    autodetector = MigrationAutodetector
```

Comment:
Question: What is this and why do we need it? 

Is that related to Django 5.2 support ? 

If it isn't I'd prefer to have it as a separate change.

Reply:
This is related to Django 5.2. This field was added to the migrate command in Django 5.2